tests/special_coverage/test_spec_cov_camp.py

```diff
@@ -104,12 +104,24 @@ def test_query_string_resolves_to_dict_when_saving(self):
     def test_query_string_filters_null_values_when_saving(self):
         """Andrew Kos comments on tests. I however, do not."""
 
+        group = {
+            "conditions": [
+                {
+                    "values": [{
+                        "value": 'test',
+                        "label": 'test'
+                    }],
+                    "type": "all",
+                    "field": "tag"
+                },
+            ],
+        }
         data = {
             "name": "special boy cammy",
             "query": {
                 "pinned_ids": [1, 2, 3, None],
                 "excluded_ids": [4, 5, 6, None],
-                "groups": ["fun guys", None],
+                "groups": [group, None],
```

Comment:
This was previously broken but hidden b/c SpecialCoverage._save_percolator() wasn't saving inactive special coverages to percolator